MANIFEST.in

```diff
@@ -1 +1 @@
-recursive-include synse/locale *.mo
+recursive-include synse/i18n *.mo
```

Comment:
i18n means localization, correct?

Reply:
i think it technically means internationalization, but i've seen it used to mean localization. not sure why its named that, but is what it is.